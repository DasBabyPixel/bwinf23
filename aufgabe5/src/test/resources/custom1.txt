16
A,0,X,0
B,1, ,1
A,2, ,2
C,3, ,3
A,3, ,3
A,3, ,3
C,3, ,3
A,3, ,3
C,3, ,3
C,3, ,3
A,3, ,3
A,3, ,3
A,4, ,4
A,5, ,5
B,6,X,6
A,7, ,7