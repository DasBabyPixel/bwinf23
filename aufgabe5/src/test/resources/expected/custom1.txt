4
A,0,X,0
A,5, ,0
B,6,X,1
A,7, ,2
